### DISCUSSION TITLE
Relocate eufy_security-*.* files...

### DISCUSSION THREAD

**Discussion by livlif2dfullest** [fuatakgun/eufy_security#159] [Q&A] [answered]
Is there a way to have the eufy_security-*.* files be stored in a different location?  Currently they're being written to my config folder and I'd like them in a child directory so I can find my other configuration files easier.  The ones in particular are the *.ts files.

Thanks in advance.

**Comment by fuatakgun** [comment, marked as answer]
I want to deprecate file based streaming, you should use rtsp add on and enable this from configuration and you will not have any files.

https://github.com/fuatakgun/eufy_security#62-installing-rtsp-simple-server-add-on-for-faster-p2p-streaming---required
